Source/ASNetworkImageNode.mm

```diff
@@ -85,6 +85,8 @@ @interface ASNetworkImageNode ()
 
 @implementation ASNetworkImageNode
 
+static BOOL _useMainThreadDelegateCallbacks = YES;
```

Comment:
Technically this should be `static std::atomic_bool _useMainThreadDelegateCallbacks(true)` or some variant. The other code in here should be OK, since c++ defines the operators to implicitly do store/load.

In practice the risk is trivial. I approve either way – let's get this landed!